test/widget_tests/after_auth_screens/app_settings/app_setting_page_test.dart

```diff
@@ -85,19 +80,10 @@ Widget createChangePassScreenDark({ThemeMode themeMode = ThemeMode.dark}) =>
     );
 
 Future<void> main() async {
-  TestWidgetsFlutterBinding.ensureInitialized();
   testSetupLocator();
+  TestWidgetsFlutterBinding.ensureInitialized();
   locator<GraphqlConfig>().test();
   locator<SizeConfig>().test();
```

Comment:
Please restore this file. We don't need to change this file.

Reply:
Its the same as previous one @Ayush0Chaudhary

Reply:
replaced it.